### DISCUSSION TITLE
HTTP Proxy support for Openai

### DISCUSSION THREAD

**Discussion by IvanovSvyatoslav** [open-webui/open-webui#4198] [Q&A]
Hello. Openai API is blocked in my region. To access it in other applications I use HTTP proxy. But when I tried to add HTTP proxy to environment it didn't help. It routes request through proxy only when I'm checking API key in interface. In other cases, for example to fetch models, it doesn't use proxy, I see 403 Forbidden (geographical block) in logs. Am I missing something? How to make it work?

**Comment by Mushy-Snugglebites-badonkadonk** [comment]
Is this what you’re looking for?

https://openwebui.com/f/hub/openai_proxy_pipe
